cmd/load-app.go

```diff
@@ -64,4 +64,5 @@ here https://gist.github.com/jloutsenhizer/8855258.
 
 func init() {
 	rootCmd.AddCommand(loadAppCmd)
+	loadAppCmd.Flags().BoolP("broad-search", "b", false, "Search for devices using comprehensive network scanning (slower but finds more devices)")
```

Comment:
Can you move this to [root.go ](https://github.com/vishen/go-chromecast/blob/master/cmd/root.go)? I think it would be better to add this everywhere.